keyboards/handwired/videowriter/rules.mk

```diff
@@ -0,0 +1,26 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+
+BOOTLOADER = lufa-dfu
```

Comment:
I believe most Pro Micros come with Caterina bootloader.

@fauxpark, what do you think here?

Reply:
Yes, this should be `caterina` otherwise the `RESET` keycode will not work (the bootloader jump process for Caterina requires a magic number placed in a certain location IIRC).